packages/application-shell/src/components/application-shell/application-shell.tsx

```diff
@@ -2,7 +2,7 @@ import React, { SyntheticEvent } from 'react';
 import { Dispatch } from 'redux';
 import { Redirect, Route, Switch, useLocation } from 'react-router-dom';
 import { ApolloError } from 'apollo-client';
-import { Flags } from '@flopflip/types';
+import { TFlags } from '@flopflip/types';
```

Comment:
Maybe I shouldn't made a breaking change. Didn't know we use it or somebody. Well, life is...